shell.nix

```diff
@@ -0,0 +1,47 @@
+{
+  pkgs,
+  pkgsForZ3,
+  self',
+  venir-toolchain,
+  verus-lib,
+  ...
+}:
+let
+  inherit (pkgs) lib stdenv mkShell;
+  inherit (pkgs.darwin.apple_sdk) frameworks;
+  venir = import ./derivation.nix { inherit pkgs self' venir-toolchain; };
+  verus-std = import ./verusStd.nix {
+    inherit
+      pkgs
+      self'
+      venir-toolchain
+      verus-lib
+      pkgsForZ3
+      ;
+  };
+in
+mkShell {
+  packages =
+    [
+      pkgs.alejandra
+      pkgs.mdbook
+      pkgsForZ3.z3_4_12
+      venir
+      self'.legacyPackages.rustToolchain
+      verus-std
+      # pkgs.rustfilt
+    ]
+    ++ lib.optionals stdenv.isDarwin [
+      pkgs.libiconv
+      frameworks.CoreServices
+    ]
+    ++ lib.optionals (!stdenv.isDarwin) [
+      # `wrangler` is the Cloudflare CLI for deploying the build Noir docs
+      # web-site. The package is currently marked as broken on macOS.
+      pkgs.wrangler
```

Comment:
Hasn't been since October of last year, see https://github.com/NixOS/nixpkgs/commit/32abce05f165cfc4de15bb6f9c1db4b2924f32b0